package.json

```diff
@@ -8,7 +8,8 @@
     "test": "yarn run lint && yarn run jest",
     "jest": "jest",
     "jest:watch": "yarn run jest -- --watch",
-    "prepublish": "babel -d dist/ src/",
+    "build": "babel -d dist/ src/",
+    "prepublish": "npm run build",
```

Comment:
Unsure why this has been split out now? Also should be using yarn to be consistent